containers/hugging-face-inference/README.md

```diff
@@ -0,0 +1,33 @@
+# Hugging Face Models
+
+### Deploy models in Serverless Containers
+
+- Export these variables:
+
+```bash
+export SCW_ACCESS_KEY="access-key" SCW_SECRET_KEY="secret-key" SCW_PROJECT_ID="project-id" REGION="fr-par"
+```
+
+- Add/remove Hugging Face models (with `.gguf` extension) in `terraform/hf-models.json` file.
+
+- Run script to deploy multiple hugging face models using terraform workspaces:
+
+```bash
+cd terraform && bash terraform.sh -a
+```
+
```

Comment:
**issue(docs):** can you add a section on how to call one of the inference endpoints? 

If you add the endpoints as a Terraform output, you can write a command that you can copy-paste using `terraform output`. See the other examples on how to do this.

There should be a command to call the "hello" endpoint to check they are working, then ideally a command for how to get an inference decision.